store/sqlstore/channel_store.go

```diff
@@ -2714,9 +2714,12 @@ func (s SqlChannelStore) GetForPost(postId string) (*model.Channel, error) {
 func (s SqlChannelStore) AnalyticsTypeCount(teamId string, channelType model.ChannelType) (int64, error) {
 
 	query := s.getQueryBuilder().
-		Select("COUNT(Id) AS Value").
-		From("Channels").
-		Where(sq.Eq{"Type": channelType})
+		Select("COUNT(*) AS Value").
+		From("Channels")
+
+	if channelType != "" {
+		query = query.Where(sq.Eq{"Type": channelType})
+	}
```

Comment:
Was this causing the miscalculation?

Reply:
Yes, this one for channel store. Because previously we only indexed public channels, but now we index private channels as well. And then there was one additional bug for the users count where we were ignoring bot users. That fix is in the EE PR.

Reply:
Thanks, nice one 👍